README.md

```diff
@@ -8,18 +8,23 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/ethereumremix?style=social)](https://twitter.com/ethereumremix)
 
 # Remix Project
+**Remix Project** is a rich toolset including Remix IDE, a comprehensive smart contract development tool. The Remix Project also includes Remix Plugin Engine and Remix Libraries which are low-level tools for wider use.  
 
-**Remix Project** is a platform for development tools that use a plugin architecture. It encompasses sub-projects including Remix Plugin Engine, Remix Libraries, and of course Remix IDE.
+## Remix IDE
+**Remix IDE** is used for the entire journey of contract development by users of any knowledge level. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs.  The IDE comes in 2 flavors and a VSCode extension:
 
-**Remix IDE** is an open source web and desktop application. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs. Remix is used for the **entire journey of contract development with [Solidity language](https://soliditylang.org/)** as well as a playground for learning and teaching [Ethereum](https://ethereum.org/).
+**Remix Online IDE**, see: [https://remix.ethereum.org](https://remix.ethereum.org)
 
-Start developing using Remix on browser, visit: [https://remix.ethereum.org](https://remix.ethereum.org)
+:point_right: Supported browsers: Firefox, Chrome, Brave. No support Remix's use on tablets or smart phones or telephones.
```

Comment:
we should specify the version number too

Reply:
@yann300 - So in checking with Circle - we test with chrome v101.0.4951.64 and with FF v100.0.1. and Brave is Chromium - so what do we say about Brave?  And by specifying version num, then we need to update this readme every we use a newer version of the browsers in a CI.  Alternatively, we can say we test with the latest version of all Firefox & Chrome ( and leave out Brave).

Reply:
we can update now the md to reflect the current version used in CI. For the moment let's not mention Brave.